docs/content/concepts/components.mdx

```diff
@@ -24,7 +24,7 @@ To learn more about the tokenomics of Sui, see the following topics:
 
 - [Sui Tokenomics](./tokenomics.mdx)
 - [Gas in Sui](./tokenomics/gas-in-sui.mdx)
-- [Gas Pricing](./tokenomics/gas-pricing.mdx)
+- [Gas Pricing](./tokenomics/gas-in-sui.mdx)
```

Comment:
Needs entry in vercel.json. I can't quite tell if this is the only URL changing, but you know where I'm going...